Test/Unit/Helper/ConfigTest.php

```diff
@@ -1224,12 +1228,14 @@ public function getAllConfigSettings()
             ['should_minify_javascript', 'true'],
             ['capture_merchant_metrics', 'false'],
             ['track_checkout_funnel', 'false'],
+            ['show_cc_type_in_order_grid', 'false'],
```

Comment:
Is this from something else or is this actually for this PR?

Reply:
Actually a couple of them were missing as well as checking the full array, see below.